documentation/openapi.json

```diff
@@ -49,9 +49,7 @@
             "description": "Process ID used to identify the target process.",
             "schema": {
               "type": "integer",
-              "description": "Process ID used to identify the target process.",
-              "format": "int32",
-              "nullable": true
```

Comment:
I believe this type of change is due to an upgrade from OAI 3.0 to 3.1, when billable was largely removed, even though the doc still says it's 3.0.1. I'm okay with this so long as the doc OAI version is corrected and we have a way of sticking with OAI 3.0.X for the `release/6.x` and `release/6.1` branches.

Reply:
Maybe back out this version change so we can investigate it separately. I'm concerned it is introducing breaking changes in the OpenAPI doc even though it claims to be the same specification version.

Reply:
I've reverted `SwashbuckleAspNetCoreSwaggerGenVersion` to `5.6.3` - this appears to have undone the OpenApi changes, but this version won't be considered compliant for the review since it's over 18 months old (9/23/2020)